pandas/tests/computation/test_eval.py

```diff
@@ -1865,6 +1865,16 @@ def test_invalid_engine():
         pd.eval("x + y", local_dict={"x": 1, "y": 2}, engine="asdf")
 
 
+@td.skip_if_no_ne
+def test_numexpr_option_respected():
+    # GH 32556
+    from pandas.core.computation.eval import _check_engine
+
+    with pd.option_context("compute.use_numexpr", False):
```

Comment:
can you add a test similar to https://github.com/pandas-dev/pandas/issues/32556#issuecomment-623364261

that we can check from a user perspective that this is working.

Reply:
added